packages/grafana-data/src/transformations/transformers/filterByValue.test.ts

```diff
@@ -12,18 +12,10 @@ import {
   FilterByValueType,
 } from './filterByValue';
 import { DataTransformerID } from './ids';
+import * as utils from './utils';
 
-let transformationSupport = false;
-
-jest.mock('./utils', () => {
-  const actual = jest.requireActual('./utils');
-  return {
-    ...actual,
-    transformationsVariableSupport: () => {
-      return transformationSupport;
-    },
-  };
-});
+const mockTransformationsVariableSupport = jest.spyOn(utils, 'transformationsVariableSupport');
+mockTransformationsVariableSupport.mockReturnValue(false);
```

Comment:
The previous method of mocking no longer worked with node_modules, so instead just rely on a classic spy to mock the return value.